### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by jonmadison**
Added scope to package name to play nicely worldwide. :)

https://github.com/microsoft/autogen/issues/2214

## Checks

- [x] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [x] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [x] I've made sure all auto checks have passed.

### CODE REVIEW

**Comment by jonmadison** [comment]
added scoping to this package; borrowed from existing microsoft project convention (`office-js`).

File: website/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "website",
+  "name": "@microsoft/autogen-website",
```
Comment (jackgerrits) [inline website/package.json:2]: Considering this package is not published can we just remove the name and version?

Comment (ekzhu) [reply website/package.json:2]: Agree, this is package is not published. Also suggest instead of @microsoft/autogen-website just autogen-website

**Review by gagb — APPROVED** [review]

**Comment by jackgerrits** [comment]
Closing since we haven't heard from you - please feel free to reopen if you'd like

**Pull request closed without merging**